src/main/scala/cromwell/engine/CallExecutionActor.scala

```diff
@@ -70,6 +71,9 @@ class CallExecutionActor(backendCall: BackendCall) extends Actor with CromwellAc
   def withRetry(work: Future[ExecutionHandle], onSuccess: ExecutionHandle => Unit, onFailure: => Unit): Unit = {
     work onComplete {
       case Success(s) => onSuccess(s)
+      case Failure(e: GoogleJsonResponseException) if e.getStatusCode == 403 =>
+        logger.error(e.getMessage, e)
+        throw e
```

Comment:
I'm always leery of transient errors regardless, it'd be nice if our "this is 99.9999% fatal" could be retried just for a brief time and "something weird happened" got retried for a much longer duration